### ISSUE TITLE
GeoJSON and fill cells

### ISSUE THREAD

**Issue by pedro-andrade-inpe** [TerraME/terrame#1219]
Check if GeoJSON can be used to create a Cellular Layer and to fill cells.

**Comment by pedro-andrade-inpe** [comment]
Test it by changing the script available at:
https://github.com/TerraME/terrame/blob/master/packages/terralib/data/itaituba.lua

**Comment by hguerra** [comment]
The script below uses GeoJSON file to create a layer and to fill cell, however the data is exported in Shapefiles.

```lua
import("terralib")

proj = Project{
	file = "itaituba.tview",
	clean = true,
	deforestation = filePath("desmatamento_2000.tif", "terralib"),
	altimetria = filePath("altimetria.tif", "terralib"),
	localidades = filePath("Localidades_pt.geojson", "terralib"),
	roads = filePath("Rodovias_lin.geojson", "terralib"),
	setores = filePath("Setores_Censitarios_2000_pol.geojson", "terralib")
}

cl = Layer{
	project = proj,
	name = "cells",
	clean = true,
	file = "cells.shp",
	input = "setores",
	resolution = 5000
}

cl:fill{
	operation = "average",
	layer = "altimetria",
	attribute = "altim",
	clean = true,
	output = "altim"
}

cl:fill{
	operation = "coverage",
	layer = "deforestation",
	attribute = "defor",
	clean = true,
	output = "defor"
}

cl:fill{
	operation = "distance",
	layer = "roads",
	attribute = "distr",
	clean = true,
	output = "distr"
}

cl:fill{
	operation = "distance",
	layer = "localidades",
	attribute = "distl",
	clean = true,
	output = "distl"
}

cs = CellularSpace{
	project = proj,
	layer = "distl"
}

m = Map{
	target = cs,
	select = "altim",
	slices = 10,
	color = "Blues"
}

m = Map{
	target = cs,
	select = "distl",
	slices = 10,
	color = "Reds"
}

m = Map{
	target = cs,
	select = "defor_0",
	slices = 10,
	color = "Greens"
}

m = Map{
	target = cs,
	select = "defor_254",
	slices = 10,
	color = "Greens"
}

m = Map{
	target = cs,
	select = "distr",
	slices = 10,
	color = "Reds"
}
```

**Comment by pedro-andrade-inpe** [comment]
@hguerra, please try with beta-3, as the output and clean arguments were removed.

**Comment by hguerra** [comment]
TerraView 5.1.3 can use GeoJSON as input to create a Cellular Layer, but the output is always Shapefile.

**Comment by avancinirodrigo** [comment]
This will be resolved by #2179
